src/XIVLauncher.Common.Unix/UnixDalamudRunner.cs

```diff
@@ -74,43 +74,57 @@ public UnixDalamudRunner(CompatibilityTools compatibility, DirectoryInfo dotnetR
         launchArguments.Add(gameArgs);
 
         var dalamudProcess = compatibility.RunInPrefix(string.Join(" ", launchArguments), environment: environment, redirectOutput: true, writeLog: true);
-        var output = dalamudProcess.StandardOutput.ReadLine();
 
-        if (output == null)
-            throw new DalamudRunnerException("An internal Dalamud error has occured");
-
-        Console.WriteLine(output);
+        // Proton-based wine sometimes throws a meaningless error, as does the ReShade Effects Shader Toggler (REST).
+        // Skip up to two errors (or any two other non-json lines). If the third line is also an error, just continue as normal and catch the error.
+        string output;
+        int dalamudErrorCount = 0;
+        do
+        {
+            output = dalamudProcess.StandardOutput.ReadLine();
+            if (output is null)
+                throw new DalamudRunnerException("An internal Dalamud error has occured");
+            Console.WriteLine("[DALAMUD] " + output);         
+            dalamudErrorCount++;
+        } while (!output.StartsWith('{') && dalamudErrorCount <= 2);
```

Comment:
This is a very crude check for valid JSON we should try to parse lines until they can be deserialized to a `DalamudConsoleOutput`, something like https://github.com/marzent/FFXIVQuickLauncher/commit/f50acf410700c1c7ee02e8bd9705e5fe8dddb58c would be better IMO...
Although I generally dislike both of these solutions that issue should really not be happening in the first place, but I guess some configurations are special